server/sonar-server/src/main/java/org/sonar/server/platform/BackendCleanup.java

```diff
@@ -56,6 +56,7 @@
   private static final Map<String, TableCleaner> TABLE_CLEANERS = ImmutableMap.of(
```

Comment:
`BackendCleanup#clearDb` is called in MediumTest between test method calls.
Deleting the default group creates an illegal content in the DB which makes the second test method call fail